cli/integration-test/integration/code_action_test.clj

```diff
@@ -48,6 +48,9 @@
         :kind "source.organizeImports"}]
       (lsp/request! (fixture/code-action-request sample-file-name 5 4)))
 
+    (lsp/mock-response :workspace/applyEdit {:applied true})
+    (lsp/mock-response :window/showDocument {:success true})
```

Comment:
These are required ... without them, the server waits forever for the client to respond to applyEdit, and never sends showDocument.